R/ds.R

```diff
@@ -0,0 +1,115 @@
+# DS
+
+#' Disposition Dataset
+#'
+#' A SDTM DS dataset from the CDISC pilot project
+#'
+#' @source \url{https://bitbucket.cdisc.org/projects/CED/repos/sdtm-adam-pilot-project/browse/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/tabulations/sdtm/ds.xpt}
+#' "ds"
+
+library(dplyr)
+library(labelled)
+library(tidyselect)
+library(admiral)
+library(admiral.test)
```

Comment:
please remove the dot - we changed our naming convention so this package is admiraltest now

Reply:
Updated